cpp/ps0006/problem 7.cpp

```diff
@@ -43,7 +43,7 @@ int main(void)
     counter(-8, 31, -2); // Error: can't start from 31 to -8 with step -2
     std::cout << "===============" << std::endl;
```

Comment:
- [x] counter gives error message for invalid value of `step`
- [x] up counter gives error message for negative value of  `step`  
- [x] down counter gives error message for positive value of  `step`
- [ ] counter gives error message to count from a number to the same number unless `step==0`